docs/source/processors/scratch.rst

```diff
@@ -25,6 +31,11 @@ You can test the output of your Scratch block text at
     (for example: Django) to create the images.
     See :ref:`accessing-scratch-image-data` section below.
 
+.. note::
+
+    The following examples assume usage of the fenced code extension, by having
```

Comment:
This needs to move to the top of the file (probably line 6) as the first example uses fenced code example.